### ISSUE TITLE
Malformed UTF-8 data modifying an item

### ISSUE THREAD

**Issue by gmontequi** [nilsteampassnet/TeamPass#3301]
### Steps to reproduce
1. Edit an item
2. Save

### Expected behaviour
Modify item without errors.

### Actual behaviour
When push save button:

_Next error occurred
Error: Malformed UTF-8 data

Raw answer from server:
﻿﻿﻿﻿﻿eyJjaXBoZXJ0ZXh0IjoiY2RJN2pLbnZ3YzUrNVNFckdsZ2M2cVIwMUwrVXZDeFVVVTlMaCtvempJdz0iLCJpdiI6IjVlOTM3NWUyMjJmYTBkOGMyZGYwNDgxYjdmYzdkNTU1Iiwic2FsdCI6ImFlYzQ5MmQzZmMxNDdlZGY3Yzk1NTM1MDM2MjYyOThjNGQ4YmRjOGFlYTY5MmIyMzVhYzlkYTk0ZWZmYzFiNGE1YzBiOTA5NmY5YTE5M2Y2NDA1OGE1Y2NiYjJjNzg0NDBlYTBhNDFkNmFhYWE0ODdkNDBiNzIzN2E4NTkwYmY3MGIwMWYzNmFmMWU2OTU2NDBiNDdlYmI3Njk1NTFiMGE2MDRlM2EzYTc3ZGFmYzA0MTRhZjNiYmNmZjVkZDk3ZmE0ZmQ2MmIzNWU5NDBlNTVlZWMxYzgxNWVhMDczM2E2YTViOTczMmI4MjM5MTFlMThiMzAxN2NmMmZhMjA5ZGQ1ZTY3NzhmMGE2YTQ1YmM5YmZiNmUzNGYxYzM4ZjU0N2U3NTMzNzE2NzJmNDM3Nzc5MjI4N2VlOGY3YTQ4ZTcxOGFjMjE0ZWUxOWMwMjZhZjAxNzYxM2U3N2E3NjcwOTczMjZmY2RhZDJiZGZkYTMwMGE0OTUyMDliOTg1YTAwNWEyNjkwY2RmYzI2YmY0NDZmYThmZmFjMjQxZGQxZTU2Mzc0YjAzMjdjY2M1MmY1ZGZmZDU2MDBlMTIzMTA1Y2RjZTAyOTZkZWJlZGRlNDY1OGIwOWRhYWUwOGY4YThjMzg4MDIwYjEzMjM0NDg5YTQ5N2YzMTViYzRlNzMxNTFkIiwiaXRlcmF0aW9ucyI6OTk5fQ==_

But, the item is saved with the modification. Trying to edit it again:

Next error occurred
Error: Malformed UTF-8 data

Informations:
- File: items.queries.php
- Function: load_item_history

Raw answer from server:
﻿eyJjaXBoZXJ0ZXh0IjoiWFAzditibG42MVFxRDFxK2llcitIaXc1S1A1bmxpUENNSWVYQ1wvRmxTUldkZnlvM3Vra3dmNVdkZ1dyMitiMFNtSVJmTmVoM3k0TjlTNVdQbE03QkIyZnNGWTJmQlNGM1hNOWoxN0hCWm5CWU5JeDhJank0RmRlYkFZS0pOWG85enMyakJTXC81Q1psUjhrSWlPTUVzMWlPS0lXMUNQSHhHXC9VYWNmZlRBTXc5WTh5dmxST0hnUm9tZFhuYmt3ZDRsVU9mTVRQV1hUQWxBT3R1R3BBbmZEUm05Sjdua2pjeFZrT2pTeGZzQjJDR0VSeTl5OUdOZExMakRIaHF6c1RcL3h4Q05FUFJIcXEya2gwWll6Uk96RWVcL0ZNejZRY2ttRmtYV3RtYVg4MmxMRVBwMzBLMXR2WWZvaXgzNXBGMFBlbzBSVXNDWjRva25nS1pMQkJ5RUN2Q2FHMlIxSllHXC8rNDgzUHpkN2RuTG1vOVk0cnVHUjh2Z0IyRnp0YmhHWEhIenhkbzAycnJKTXNyT0p6V3oweE96U3JDQ1haM2g2eXkrNHNjUW5lWnd6MXV6UWk0aU1NVEZHSEhjd1lZYnl5MG9wQVZkRWk1R1wvV21lcTNhUWtFU3JVY3hzd0E0d204V3RuQ2JCQndEQmN6Y2p2N1E4R3NsVVA1d3lETExGUmFoTUE2dUxBNHVXbER0UksyU2crXC9jdk1LTFhDYW5RNytpV1Jnb2RsVDJqSTlmNDkwbE0yQVwvY3ZnblNickRycitpQ0IxXC8xbGc0WklYbUEyZnpsMjRMbXM1WTluK0Irb0llSnNpeEFnZTM4ZFFXNlJwZTZCbmE1QWdkZERKTVc1WGhtVlFNTGNYdUlIM0xLd0ROWUI2TW1UZmVwXC92cFVweldqdGFMOWdNMDhXTVhYMkNyQ2o1cTJaQUhDdysrUUNyaDM0RzExXC9HU0dQaUxDOXpmRkdoWEpaSHFtNlpGTityN1BNQnpZVFFtM1J2UVZtTXJpdFNVRUtDSmpZQm9qcVwvUkdkUEtkRGU5RjZWUXllMGhKZ0RLU2h0RFBsOG5hTGZFejV2VnJSNVhHQW4reWpMdDNmallva2VlS1pGXC9yTW9GSzFXYXMyR2JRYzNaZmh2cm1NOWNoMDd1bzFzVGZHYkZPbkUwNWRXbk1lUkFSbDFHM25FazFhWk1iOTVkM0xNcG5qRUw1NDhHaWN4THJ0T3IyT05BSjZTekc1a2tzTVwvRHpcL1V3MGJYUEFKYWx5eFpIdUdTd3lXSDNrV3poRzhpSFkzcnFKRFEwUFkwSlJkNnlFZ3JpUTZrY05ycFZ0d2hSWVQ4WFM2QVBvZ3R3RENhUkFkZk8yQ2ZlZ1lnM3pEaEVmZ3hKdTFcL01jUEN3UnRQRUd4aDRDaDZuWDdtaVMwUUpmY2F2VTRNT2pEWFc2VHNGUmlvTjlOOUdMZXoxRVYrc1IrM2tTbUw0M3RLd3N1bzVZejRKNUxQMnJudkpvNUhDb0ZkejM2Sk5Sc2tJd05IaHVvWFY2dWdIcDhVdkhtYjJaWmo1YU5tYnFOc3B6NHJWajltOVwvSFl4SVwvcWIrM1V4c2xkR3pvUWJ6MEtQS29SaHNJMUlDYkhGbHNhQzBkZzV2b25GSjhTK2RIXC9ITTNNelVldnkxQ3hWTFZsdGQxUENNUDVHRjVDZUsyZ2FqMXFIUmNpbGlYSHBMSlFJMTBzWHdLTGU1M09JMDVMSVlwWnBoMnF5ZVB6VVdiTDdcL2V4a3hQVW1rM3U0N3VxOXdiWW5Qbk84QXJBOWZKMTRKcVJFeFBJRUpWZGJrUTZZSURCek94UytvWkNkNmpnWWxrajJwN3dFN1Z5eXYwU0F6XC9BXC8wTnpEVG1cL21oQmlPNGYzUlwvMmtKdUNOQ3N0ME40OVZSWEhoZ0piY25MYUJRYXBMY253TUVsSk4zSWFIVkFrRWFaanN5b2h1OTFtanlqRkd1N3V4cUFBQnJmUDhDQVBUcEp4VWFOQT09IiwiaXYiOiIzYjc2ZTExYTE4YzUxNDcyZDZkNjI5Y2QzMjAwNzQ2MiIsInNhbHQiOiI0ZDhiMTIzMjA2MmJjOTFjZDIxN2QxN2Q0YjEwMmQ2MjJiMmUxNDgyNzVmMmQ2NTM4OTIzMzg4NTQ2MjQ2YTY1ZTMzYTQ4ODEyNWVkOTUxNTg1ODBmMTgwODc4ZjJlN2MyNTFjYWI1MDQwNDI0Mjc0ZWM4YzdiNzNmMWZhZDk1NmM1M2YxNzE0OGI4ZDU0YmU0NzViOTRmMjA4ZjU2YTUyODBkY2FlZjY1ZjU1NTIyNTcxY2VkMmRmYWI3ODY1M2Q0Zjc1YzBiY2RiNjljZjNhZjM3MThiNTQyNjRmZTRjZDViMDZkYWMwMTZlNjdhMDQ4Njc1MjI1YTZlZDhiYjlhODJjM2E1ZDAwY2Y3YWM4YWJiMzE4NTIyYWVjMGVhM2NiZjAyNTNmNTdjNmIwODMxNzBmNzNlOTVjNTVjNzYyYjFkZWEyZTYwYTBlMzRiOWEyNmQxOTJlNTQ0YmMwZmU0NDJlNDA2ODMzNDgxMjUzYjhiODQ1ZWEwZjQyOTg4OGMzMjQyY2YxZDEzNWMyN2M5ZGYyYTE2MzA0ZTA3MjQ3YmYzMjIwOGViMmE0MDkzMmY4N2JmNDMzZTFiOThhMzRlMTY0M2E4MjQ3YzQ0N2M0NWUzZmI3NTQzM2U2MmMzYTJmYjNhOWQyZTgyZjA0YjZlY2MwN2M4NWY5ZWE2M2VmZSIsIml0ZXJhdGlvbnMiOjk5OX0=

And again:

Next error occurred
SyntaxError: Unexpected token '', "{"ci"... is not valid JSON

Raw answer from server:
﻿﻿﻿﻿﻿﻿﻿﻿eyJjaXBoZXJ0ZXh0IjoiU2Y2d2kzK3JsY0JDT0lBazY5Y0RoMjVETEtHRUhlRENDV3JielVWNFE5RT0iLCJpdiI6Ijk3ZGRmOTY2YTE5MjQzZTUwODNkMjdkOGUxNmJjNDgwIiwic2FsdCI6ImFjNjY5ZTlhZDkwMGIxNmJiZWEzZDMyMGZkYTlmZWE4OGRmOGZhNzM4NWUyY2M3OGQyMWNiMWQxMDk3Y2I3MjAyN2VhMjNkMDg1NDJlODExNzZkOWNkYWI3ZWZjNTY3MWUxZjAzOWE0NmQ5MzczYzNmNmE0ZmFiMzgwMDhmYzdmYmIyMjYwNzMzZTkwNjkyYTRkMWQ4MTRiMWM4N2FiMTMxMjMxYTE1Mjg1MTliM2Y3OWQxY2Q1YTJjZmExNGZlZGQ3YTgwNTJhOTA1MWViNTAzMDcyYjA2ZjRkYmJmYTk2ZWNmNGJhNmI1MjNiZGM0OWY0YzFmODI1ZDhiNTQzZGFlZTlhMWMwNjE4YmRmODAzNjIyMWM4OTlhNzllZWU1OTM1MWFlYjI4Yzk3M2E3ZjIzM2FlMTgyMjEyZTY4NWZlMDI0MTUzYzhjNjViMmM1YzJjYjEwNjc4NzI2ZTJiYmQ2NzRjZTk3YzEzNGY0MmIzNjRjZjM3YmViNWE0OWJkNGYyNzA5OGViOTg0OTc1NGRkYzUwMGVhMzZiYmY4MzkxYWE2ZmNiOGY2NzY1NjIwOTQ5ZGE2M2MyYTM4ZDI3NDVhYTFiNWI5NTkzYjUwYTY2NzM1YWM2NWRiZWZkNTc5N2U0YWZlYTA1Y2IyMjA2NDFhNWViMGEyNmE5MDAyMDcxIiwiaXRlcmF0aW9ucyI6OTk5fQ==

### Server configuration
**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**Teampass version:** [dd90cca](https://github.com/nilsteampassnet/TeamPass/commit/dd90cca8cf39a3340ecb2bcf6c6098a737db928b)

**Teampass configuration file:**

**Updated from an older Teampass or fresh install:**
PLEASE attach to this issue the file `/includes/config/tp.config.php`.

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
```
Insert your webserver log here
```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Insert the log here and especially the answer of the query that failed.
```

**Comment by nratineau** [comment]
@gmontequi Had some issues with the items as well due to the language function. 
Submitted a PR.
Cannot say you have the exact same problem but if you have a test environment and want to try it out: https://github.com/nilsteampassnet/TeamPass/pull/3302
Specially the commit: https://github.com/nilsteampassnet/TeamPass/commit/e814c19109db86f9ae831a906528b303d02dcb16
Caution, this is not merged to the main branch and I don't know it would ever be merged :)

**Comment by gmontequi** [comment]
Same problem with [ccfb790](https://github.com/nilsteampassnet/TeamPass/commit/ccfb790d439d2013a98306608c3e877c4ffa1304)

**Comment by gmontequi** [comment]
Thank you @nratineau 
I have tried it but same error :-(

**Comment by nratineau** [comment]
@gmontequi Can you test with the latest commit. If you still have the issue, can you provide
- Your _tp.config.php_ file (make sure to remove any login/password for your LDAP, email,...)
- Operating system:
- Web server:
- Database:
- PHP version:
- Log from the web-browser developer console (CTRL + SHIFT + i)
- Web server error log

Those will help dig further into what's happening.

**Comment by gmontequi** [comment]
> @gmontequi Can you test with the latest commit. If you still have the issue, can you provide
> 
> * Your _tp.config.php_ file (make sure to remove any login/password for your LDAP, email,...)
> * Operating system:
> * Web server:
> * Database:
> * PHP version:
> * Log from the web-browser developer console (CTRL + SHIFT + i)
> * Web server error log
> 
> Those will help dig further into what's happening.

Solved with [f22899f](https://github.com/nilsteampassnet/TeamPass/commit/f22899f86e8ef3d0301d664b5e240e70725120b3)

Thank you so much! Great job!
